command/events.go

```diff
@@ -44,7 +46,15 @@ Usage: vault events subscribe [-format=json] [-timeout=XYZs] eventType
 
 func (c *EventsSubscribeCommands) Flags() *FlagSets {
 	set := c.flagSet(FlagSetHTTP)
-
+	f := set.NewFlagSet("Subscribe Options")
+	f.StringSliceVar(&StringSliceVar{
+		Name: "namespaces",
+		Usage: `Specifies one or more patterns of namespaces to subscribe
```

Comment:
I think it's important for the integrity of the namespaces feature more generally that we should only be able to subscribe to namespaces that are equal to or children of the request's namespace. We should probably always subscribe to the request's own namespace, and treat the patterns here as additional child namespaces to subscribe to?

```suggestion
		Usage: `Specifies one or more patterns of child namespaces to subscribe
```

Reply:
Ah, I think I like that.